src/gui/powermanagement/powermanagement.h

```diff
@@ -28,38 +29,39 @@
 
 #pragma once
 
-#include <QObject>
+#include <QtSystemDetection>
 
-#ifdef Q_OS_MACOS
-// Require Mac OS X >= 10.5
-#include <IOKit/pwr_mgt/IOPMLib.h>
+#if defined(QBT_USES_DBUS)
+#include "inhibitordbus.h"
+using InhibitorImpl = InhibitorDBus;
+#elif defined(Q_OS_MACOS)
+#include "inhibitormacos.h"
+using InhibitorImpl = InhibitorMacOS;
+#elif defined(Q_OS_WIN)
+#include "inhibitorwindows.h"
+using InhibitorImpl = InhibitorWindows;
+#else
```

Comment:
Isn't it supposed to use the native method on Windows and macOS, even if `QBT_USES_DBUS` could be defined there for some reason?
```suggestion
if defined(Q_OS_MACOS)
#include "inhibitormacos.h"
using InhibitorImpl = InhibitorMacOS;
#elif defined(Q_OS_WIN)
#include "inhibitorwindows.h"
using InhibitorImpl = InhibitorWindows;
#elif defined(QBT_USES_DBUS)
#include "inhibitordbus.h"
using InhibitorImpl = InhibitorDBus;
#else
```